views/base.html

```diff
@@ -3,7 +3,12 @@
    <head>
        {% include 'meta.html' %}
       <title>SoA</title>
-      <link rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css">
+      <script type="text/javascript">
+        function swapStyleSheet(sheet){
+             document.getElementById('pagestyle').setAttribute('href',sheet);
```

Comment:
Instead of just changing the href shouldn't we be creating a full new element and replacing the existing one?

Reply:
Yeah, I looked there but couldn't get a lot out of it. 
Ohk, let me see how we can create new element instead of replacing.

Reply:
even sakura demo isn't managing the user theme. it comes to normal if you move to another page or reload.